content/en/docs/instrumentation/python/manual.md

```diff
@@ -246,3 +271,73 @@ set_global_textmap(B3Format())
 ```
 
 Note that environment variables will override what's configured in code.
+
+## Metrics
```

Comment:
Maybe it's worth linking to the language-agnostic document here as well? https://opentelemetry.io/docs/concepts/signals/metrics/

There's basically nothing there right now but that should change in the coming months.